requirements-test.pip

```diff
@@ -1,3 +1,4 @@
+freezegun~=1.0.0
 pytest>=2.9.0
 pytest-cov~=2.10.1
-urllib3-mock~=0.3.0
```

Comment:
This library had a last release some 4.5 years ago... replace it with responses and change the corresponding usage.